nodejs/week2/MealSharingWebsite/src/backend/routes/maxPriceRoute.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express.Router();
+
+const [filterCheapMeals , filterHighCapacityMeals , giveOneRandom] = require('./mealsEvaluator.js');
+
+const addReviews = require('./addReviews.js')
+
+const meals = require('../data/meals.json');
+const reviews = require('../data/reviews.json');
+const reservations = require('../data/reservations.json');
+
+app.get("/maxPriceRoute/:price" , (req , res)=>{
```

Comment:
It's great, it works! :) However in the homework it was suggested to make it with query parameters after '/meals'.
So we should get the same results using '/meals?maxPrice=90'.